src/lib/tooltipDescriptions.ts

```diff
@@ -14,7 +14,9 @@ export enum TOOLTIP_DESCRIPTIONS {
   ORDER = "The amount sold/bought. Also surplus for this order. This is the positive difference between the initial limit price and the actual (average) execution price.",
   ORACLE_TOKEN_SELL = "The oracle that will provide the price of the token being sold to the smart-contract. Both oracles must have the same quote currency.",
   ORACLE_TOKEN_BUY = "The oracle that will provide the price of the token being sold to the smart-contract. Both oracles must have the same quote currency.",
-  RELATED_ORDERS = "The orders created based on your initial request",
+  RELATED_ORDER = "The order posted on the CoW orderbook that is related to this stop loss order.",
```

Comment:
```suggestion
  RELATED_ORDER = "The order posted on the CoW Orderbook that is related to this Stop Loss order.",
```

what does this mean though?

Reply:
Basically the order uid